src/tasks/batch_integration/workflows/process_datasets/main.nf

```diff
@@ -1,7 +1,7 @@
 workflow auto {
-  findStates(params, config)
+  findStates(params, thisConfig)
     | run_wf
-    | publishStates([:])
+    | publishStates([key: thisConfig.functionality.name])
 }
```

Comment:
```suggestion
workflow auto {
  // TODO: `thisConfig` might be renamed to `meta["config"]` in the future
  findStates(params, thisConfig)
    | process_datasets.run(
      auto: [publish: "state"]
    )
}
```

Reply:
this seems like a weird suggestion to me.
Als gives an error:

```bash
Error in module 'process_datasets' id 'pancreas/log_cp10k': required input argument 'id' is missing. Expression: (inputs.containsKey(arg.plainName) && nextflow.ast.LangHelpers.compareEqual(inputs.get(arg.plainName), null))

 -- Check script 'target/nextflow/batch_integration/workflows/process_datasets/main.nf' at line: 631 or see '.nextflow.log' file for more details
 ```